core/server/api/settings.js

```diff
@@ -52,6 +51,9 @@ updateConfigCache = function () {
         }
     }
 
+    // @TODO: why are we putting the settings cache values into config?we could access the cache directly
+    // @TODO: plus: why do we assign the values to the prefix "theme"?
+    // @TODO: might be related to https://github.com/TryGhost/Ghost/issues/7488
```

Comment:
I will add a task to https://github.com/TryGhost/Ghost/issues/7488 when PR was merged 👍